src/column.rs

```diff
@@ -395,10 +423,18 @@ impl Column {
 					tables.index.validate_plan(record.index, log)?;
 				} else if let Some(table) = reindex.queue.iter().find(|r|r.id == record.table) {
 					table.validate_plan(record.index, log)?;
-				}
-				else {
+				} else {
+					// [Review] could this case be handle by triggering reindex in 'replay_all_logs',
+					// or is it specific to validate?
```

Comment:
We need to to start re-indexing during validation, so that validation can continue.